src/store/auth/reducers.tsx

```diff
@@ -0,0 +1,37 @@
+import {
```

Comment:
If you don't use React JSX syntax, I don't recommend using jsx/tsx extensions — this might be a) confusing, b) cause parsing errors in rare cases.

Reply:
I totally agree with you!
Fixed